setup_container.py

```diff
@@ -37,6 +38,8 @@ def create_venv(name: str, path: Path, interpreter):
     new_env["PATH"] = abs_str(venv_path / "bin") + ":" + new_env["PATH"]
     subprocess.check_call([interpreter, "-m", "venv", abs_str(venv_path)])
     subprocess.check_call([abs_str(venv_python_path), "-m", "pip", "install", "-U", "pip"], env=new_env)
+    # todo cleanup
+    subprocess.check_call(["cd", abs_str(models_path), "&&", "git", "apply", "/app/models_gdrive.patch"], env=new_env, shell=True)
```

Comment:
use `cwd` param of `check_call` 

```suggestion
    subprocess.check_call(["git", "apply", "/app/models_gdrive.patch"], env=new_env, shell=True, cwd=models_path)
```